### DISCUSSION TITLE
Buffer file on ram disk / tmpfs

### DISCUSSION THREAD

**Discussion by SwimGeek** [transceptor-technology/siridb-server#191] [General]
Hi

We have one VM that's doing a lot of disk writes and destroying SSDs. (UISP app)

Do you maybe have a guide for using a ram disk / tmpfs to have less SSD writes. In this case I'm ok with the risk of data loss - I just have a daily backup of the VM.

I spotted a database.conf file, with a [buffer] and a path variable.

Can you point me at an example where this is moved to ram?

Any other advice?

**Comment by joente** [comment]
@SwimGeek , You can put the following line in `database.conf`:

```
path = /tmpfs
```

Next, shutdown SiriDB and move the buffer file:

```
mv <OLD_PATH> /tmpfs/buffer.dat
```

Start SiriDB again.
